layouts/community/60_ansi/stanrc85-ansi/keymap.c

```diff
@@ -54,7 +54,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 		_______, _______, LCA(KC_QUOT), KC_VOLU, LCA(KC_SCLN), _______, _______, KC_HOME, KC_UP,   KC_END,  _______, _______, _______, RESET,
```

Comment:
What you could do while you're at it is make aliases for these longer keycodes (e.g. `#define CA_QUOT LCA(KC_QUOT)` at the top of the file) and then have a nice, uniformly aligned keymap 🙂

Reply:
Ok good idea. I'll get those added